app/controllers/packages_controller.rb

```diff
@@ -3,7 +3,7 @@
 class PackagesController < ApplicationController
   before_action :set_package, only: %i[show update customer_resources]
 
-  deserializable_resource :package, only: :update,
+  deserializable_resource :package, only: %i[create update],
```

Comment:
`[:create, :update]` feels more straightforward. Is this the new rails convention?

Reply:
Without `%i` the build fails due to rubocop.